docs/src/main/sphinx/connector/druid.rst

```diff
@@ -67,3 +67,34 @@ SQL support
 The connector provides :ref:`globally available <sql-globally-available>` and
 :ref:`read operation <sql-read-operations>` statements to access data and
 metadata in the Druid database.
+
+Table functions
+---------------
+.. _druid-query-function:
+
+``query(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``query`` function allows you to query the underlying database directly. It
+requires syntax native to Druid, because the full query is pushed down and
+performed in-place. This can be useful for accessing native features which are
+not implemented in Trino or for improving query performance in situations where
```

Comment:
not available in Trino, or

Reply:
@colebow please add a comma before "or".